jsbuiltin/jsbuiltin_test.go

```diff
@@ -0,0 +1,138 @@
+package jsbuiltin
```

Comment:
The `jsbuiltin.go` file has a `// +build js` build tag. This `jsbuiltin_test.go` file should too, otherwise this package tests will fail with `go test`.